components/button/button.js

```diff
@@ -0,0 +1,109 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { ButtonMixin } from './button-mixin.js';
+import { buttonStyles } from './button-styles.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
```

Comment:
Also there's no RTL-specific CSS for buttons, so no need to apply the `RtlMixin` either.